ext/RMagick/rmimage.c

```diff
@@ -3923,7 +3923,7 @@ Image_constitute(VALUE class, VALUE width_arg, VALUE height_arg
     exception = AcquireExceptionInfo();
 
     // This is based on ConstituteImage in IM 5.5.7
-    image = AcquireImage(NULL);
+    image = rm_acquire_image((ImageInfo *) NULL);
     if (!image)
     {
         xfree(pixels.v);
```

Comment:
In additional, `(void) DestroyExceptionInfo(exception);` is required at here to avoid memory leak.

Reply:
This was removed in another PR. Will rebase this.

Reply:
Ah, i see.